piksi_tools/console/console.py

```diff
@@ -412,9 +412,9 @@ def update_on_heartbeat(self, sbp_msg, **metadata):
     self.num_sats = temp_num_sats
 
     if self.settings_view: # for auto populating surveyed fields
-      self.settings_view.lat = self.solution_view.latitude
-      self.settings_view.lon = self.solution_view.longitude
-      self.settings_view.alt = self.solution_view.altitude
+      self.settings_view.lat = self.solution_view.latitude_list
```

Comment:
Are we certain that the  object on the solution_view will never change?  i.e., the nump.pad function doesn't change the underlying object?